svcomp-benchmark/results/README.md

```diff
@@ -0,0 +1,5 @@
+Data collected from running [RV-Match](https://runtimeverification.com/match) on the [SV-COMP Benchmark Verification Tasks](https://github.com/sosy-lab/sv-benchmarks). These results were collected on April 13th, 2016. More details about the verification tasks, and the structure of the benchmark can be found at [SV-COMP's website](http://sv-comp.sosy-lab.org/2016/index.php).
```

Comment:
@msaxena2 , this text is flat, making no points and giving no data synthesis.  What are your findings?  Why should one be interested in reading this?  You should also explain how one can run rv-match on these individually, and how to use the scripts.  We want the link we provide them with to be self-contained, to need no further explanations.  I will tell them the big point in my original message, but they will pass the link to each other only and may lose track of my original message.